api/reference/1.0/notes/list/data_element.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
These list examples should follow a hierarchy of `/1.0/{RESOURCE}/list_notes.md`

Reply:
I'll reorganize them.

So, you're okay with all the `create` operations nested under Note, but you want the `list` operations listed under their Notable? I covered a lot of permutations in my PR comment, but that ain't one of them.

So...
* the `create` ops create Notes, so they belong under Notes
* the `list` ops are accessed through a <notable>, so they belong under their Notable

Is that the idea?

Reply:
That's the idea, for now.